### PR TITLE
Add storage policy details in the detail view

### PR DESCRIPTION
**Pull request by Pearl1594**
Adds storage policy details in the detail view tab of a Disk and Compute offering

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by borisstoyanov** [comment]
LGTM
Screen Shot 2020-10-09 at 13 02 49

File: src/config/section/offering.js [hunk 1]
```diff
@@ -111,7 +118,13 @@ export default {
       permission: ['listDiskOfferings', 'listDomains'],
       params: { isrecursive: 'true' },
       columns: ['name', 'displaytext', 'disksize', 'domain', 'zone', 'order'],
-      details: ['name', 'id', 'displaytext', 'disksize', 'provisioningtype', 'storagetype', 'iscustomized', 'tags', 'domain', 'zone', 'created'],
+      details: () => {
+        var fields = ['name', 'id', 'displaytext', 'disksize', 'provisioningtype', 'storagetype', 'iscustomized', 'tags', 'domain', 'zone', 'created']
+        if (store.getters.apis.createDiskOffering.params.filter(x => x.name === 'storagepolicy').length > 0) {
```
Comment (rhtyd) [inline src/config/section/offering.js:123]: @Pearl1594 can you add check what if the API doesn't exist or is available?

File: src/config/section/offering.js [hunk 2]
```diff
@@ -29,7 +30,13 @@ export default {
       permission: ['listServiceOfferings', 'listDomains'],
       params: { isrecursive: 'true' },
       columns: ['name', 'displaytext', 'cpunumber', 'cpuspeed', 'memory', 'domain', 'zone', 'order'],
-      details: ['name', 'id', 'displaytext', 'offerha', 'provisioningtype', 'storagetype', 'iscustomized', 'limitcpuuse', 'cpunumber', 'cpuspeed', 'memory', 'hosttags', 'tags', 'domain', 'zone', 'created'],
+      details: () => {
+        var fields = ['name', 'id', 'displaytext', 'offerha', 'provisioningtype', 'storagetype', 'iscustomized', 'limitcpuuse', 'cpunumber', 'cpuspeed', 'memory', 'hosttags', 'tags', 'domain', 'zone', 'created']
+        if (store.getters.apis.createServiceOffering.params.filter(x => x.name === 'storagepolicy').length > 0) {
```
Comment (rhtyd) [inline src/config/section/offering.js:35]: We need an undefined/null check

**Comment by Pearl1594** [comment]
@rhtyd added check for existence of API

**Comment by Pearl1594** [comment]
@blueorangutan package

**Review by rhtyd — APPROVED** [review]

**Pull request merged**